### ISSUE TITLE
Explanation after changing the weight

### ISSUE THREAD

**Issue by 979736316** [lululxvi/deepxde#118]
Sorry, this may be a very simple question.When I was running the case "Burgers", the default wf and wb should be 1. When I multiplied them by 1000, I found that the final loss value is relatively small, and I can get a good calculation result. This weight Didn't it play a certain role?

`model.compile("L-BFGS-B",loss_weights = [1,1000,1000])`

**Comment by lululxvi** [comment]
The weights are defined in Eq. (2.2) in the DeepXDE paper. Using good weights leads to better results. So sometimes it is important to tune the weights.

**Comment by 979736316** [comment]
> 权重在等式中定义。（2.2）在DeepXDE文件中。使用良好的砝码可以产生更好的结果。因此有时候调整权重很重要。

Thank you very much for your quick reply. I also have a question about the adjustment rules of weights. When I run, I find that the loss displayed is in the same order of magnitude, can I think it is more appropriate to set the weight to 1.

**Comment by lululxvi** [comment]
The default weight is 1, and the loss displayed is the weighted loss, not raw loss. It is better to keep the weighted loss of the same order of magnitude.
